src/custom/hooks/useUSDCPrice/index.ts

```diff
@@ -237,15 +241,19 @@ export function useHigherUSDValue(currencyAmount: CurrencyAmount<Currency> | und
   return useMemo(() => {
     // USDC PRICE UNAVAILABLE
     if (!usdcValue && coingeckoUsdPrice) {
```

Comment:
I think this logic is unnecesarilly complicated. 

All you do is
`return coingeckoUsdPrice || usdcValue` or am i missing sth?

Reply:
Also, U would call it `gpUsdcValue` to differenciate our internal price, from the resulting USD estimation. Also for analogy with `coingeckoUsdPrice` naming convention

Reply:
added the ìf`statements for logging. i agree the `||` is much more concise and probably should be reverted to that